### DISCUSSION TITLE
zh/guide/community

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#33] [Announcements]
# zh/guide/community

AList v3的文档

https://alist.nn.ci/zh/guide/community.html

**Comment by l3056573035** [comment]
我在添加储存挂载的时候怎么报：Failed create storage in database: attempt to write a readonly database 数据库已经开放了权限  版本是V3 挂载的是Onedrive云盘

**Comment by ma5d** [comment]
换电脑了，怎样导入导出数据

**Reply by ma5d** [reply]
知道了

**Comment by l3056573035** [comment]
什么邮件 ？&nbsp;我什么也没发啊
&nbsp;



***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年11月28日(星期一) 晚上10:54
收件人: ***@***.***&gt;; 
抄送: "时光凉  ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [alist-org/docs] zh/guide/community (Discussion #33)





 
怎么老是给我发邮件啊，这个怎么取消掉。

**Comment by ghost** [comment]
我能把github挂载上去吗？

**Comment by tongtong79** [comment]
我群晖上怎么加不了天翼云，阿里云盘是可以的

**Reply by tongtong79** [reply]
V2是没问题，更新到V3后就不行，换回v2.6.4-0-gbd52df1-dirty就没问题全能正常挂载

**Comment by kevinwenyu** [comment]
怎么禁止匿名访问？

**Comment by jjh00421** [comment]
获取管理员权限时提示。。

docker exec -it alist ./alist admin

**Comment by lizt520** [comment]
现在无法复制，无法播放。怎么回事？以前还可以的。

**Comment by calvinsmart669** [comment]
群辉 Docker 登陆 用户名 和 密码 输入一直无法登陆是怎么回事？

**Comment by lizhiwei8061** [comment]
苹果手机webdav无法正常访问的问题，怎么解决呢？报错：Network request failed

**Comment by mtk2025** [comment]
有偿求加https://tmp.link/ 钛盘，q285742765

**Comment by 15109827573** [comment]
大大，我使用的docker的alist，已经映射/opt/alist/data/temp/aria2和/opt/alist/data/temp/qbittorrent两个路径，使用离线下载时出现以下情况

![This is an image](https://s1.locimg.com/2023/03/03/e7cf6562e7cf0.png)

经过排查，离线下载时，都能成功调用aria2和qbittorrent，但软件会在/opt/alist/data/temp/aria2和/opt/alist/data/temp/qbittorrent（这两个路径没有就会自己创建，是实际路径，不是映射路径）下载文件夹和文件。上传时却是找的映射路径，也就是在/mnt/sda1/aria2和/mnt/sda1/qbittorrent（映射后的实际路径）里找文件，所以就会出现上图提示的“找不到文件夹或文件”。
映射目录是根据网上教程，修改了hostconfig.json的Binds数组和config.v2.json的MountPoints，在docker管理界面直接点击路径显示映射成功，在/mnt/sda1/aria2和/mnt/sda1/qbittorrent存储文件，在docker管理界面直接点击路径也能看到该文件。到这里就不知道问题出在哪里了，请大大帮忙看看

![This is an image](https://s1.locimg.com/2023/03/03/30b862fab6efe.png)
容器管理界面能显示映射路径

![This is an image](https://s1.locimg.com/2023/03/03/3e407ed266398.png)
这是alist调用aria2离线下载后，aria2在/opt/alist/data/temp/aria2自行创建相应文件开始下载

![This is an image](https://s1.locimg.com/2023/03/03/a59f7c140d748.png)
这是实际地址/mnt/sda1/aria2，却没有文件

**Reply by PeakGymnast** [reply]
temp是自动创建的下载缓存文件，没下载完，数据还在temp文件夹缓存，等下载完了，文件就自行移动到你映射的文件夹里面去了，你不会才知道下载器下载吧。文件缓存，不见得，都是直接在下载文件夹出现，尤其是分块下载

**Comment by Ezio125** [comment]
最近挂载阿里云盘open，出现Failed init storage: Post "https://api.nn.ci/alist/ali_open/token": EOF这个错误，不知道如何解决。

**Comment by Jason880459** [comment]
我用Raidrive映射到本地的时候试了一下，天翼云客户端那个盘我只能在里面建文件夹不能往里面复制黏贴文件，说我没有权限，这是为什么呢，我alist用的是v3最新版的

**Reply by Cecilia-Zz** [reply]
请问你的问题解决了吗

**Comment by rawn123** [comment]
请问群晖docker安装的alist怎么更新升级了，必须得重装吗？

**Comment by CrazyJasonwell** [comment]
新版3.14 添加存储还是不能选到阿里云盘。。。。

**Comment by szshaojiang** [comment]
今天115突然出现权限问题，目前发现打不开视频、图片，txt文本可以打开，windows10+radrive挂载，上传图片可以但很卡，下载时先卡很长时间后报错，要S-0-1-1权限

**Comment by Winyer** [comment]
负载均衡没了吗？

**Comment by shihun01** [comment]
请问大佬挂载的网盘上传文件都是错误，或者上传不过去，可以新建文件夹，这个要怎么解决

**Reply by Cecilia-Zz** [reply]
请问你的问题解决了吗

**Comment by BestCNCoder** [comment]
想要沟通一下。按需付费需要一些新功能，权限那一块，按部门来。或者，现在好像不能给一个用户设置访问多个平级的文件夹啊

**Reply by BestCNCoder** [reply]
我的QQ和vx都是 7413755

**Comment by 15109827573** [comment]
您的来信我已收到

**Comment by LIKIQ** [comment]
./alist-proxy: line 1: syntax error near unexpected token `newline'
./alist-proxy: line 1: `<!DOCTYPE html>'
Ubuntu运行alist-proxyt报错这是因为啥

**Comment by ijchengo** [comment]
123网盘挂载挂了，，，怎么办？？

**Comment by 2080552127** [comment]
请问在挂载阿里网盘后，ppt文件不能在线打开怎么办，微软显示：File not found
The URL of the original file is not valid or the document is not publicly accessible.
Verify the URL is correct, then contact the document owner.
Learn more

**Comment by maoimiao** [comment]
docker安装最新版alist 创建好容器后 启动不成功 手动启动提示 code 304 not modefiled 请问 这个怎么解决？谢谢

**Comment by ncdz9292** [comment]
大佬，我遇见一个奇怪的问题，局域网内全设备万兆，通过SMB访问本地NAS读写速度可以达到800MB/s以上，但是我把这个NAS挂到Alist上，然后局域网通过Alist中WebDav方式访问这个NAS，速度最高就只有45MB/s，Alist安装在CentOS7 Docker上的，这是什么原因？

**Comment by 15109827573** [comment]
您的来信我已收到

**Comment by rggameffl1** [comment]
请问挂载123云盘后，打开提示：failed get objs: failed to list objs: 签名错误,请检查您的本地时间是否为东八区时间-ak illegality 999，请问该怎么解决

**Comment by jessyrow** [comment]
我的电脑是window10， 下载了alist，但是安装不了， 为什么？
D:\alist-windows-amd64>alist server
INFO[2024-01-27 17:01:40] reading config file: data\config.json
INFO[2024-01-27 17:01:40] load config from env with prefix: ALIST_
INFO[2024-01-27 17:01:40] init logrus...
WARN[2024-01-27 17:01:40] init tool qBittorrent failed: Post "http://localhost:8080/api/v2/auth/login": dial tcp [::1]:8080: connectex: No connection could be made because the target machine actively refused it.
WARN[2024-01-27 17:01:40] init tool aria2 failed: failed get aria2 version: Post "http://localhost:6800/jsonrpc": dial tcp [::1]:6800: connectex: No connection could be made because the target machine actively refused it.
INFO[2024-01-27 17:01:40] init tool SimpleHttp success: ok
INFO[2024-01-27 17:01:40] start HTTP server @ 0.0.0.0:5244

**Comment by 15109827573** [comment]
您的来信我已收到

**Reply by ma5d** [reply]
感谢，不过我已经解决了。
好像需要下载某个网页版的aria2ng。
但是由于比较缓慢，我已经不准备使用了。

**Comment by 15109827573** [comment]
您的来信我已收到

**Comment by nethuan** [comment]
本地存储，文件及文件夹超多，列表很慢怎么办？

**Comment by 15109827573** [comment]
您的来信我已收到

**Comment by zyhqwq** [comment]
在用户admin下进行权限修改时无法取消勾选任何一个权限，怎么回事啊

**Comment by Y135246** [comment]
😃

**Comment by xiaoyi0805** [comment]
为什么我挂123云盘，挂载两个，结构目录文件都一样为什么，我一个云盘能看见A文件，但是另外一个云盘缺看不见A文件是什么原因

**Comment by 15109827573** [comment]
您的来信我已收到

**Comment by Namialways** [comment]
挂载的百度网盘视频不能看了，他们app的订阅好像也不能看了

**Reply by Namialways** [reply]
已解决，弄了一个多小时没弄好，和百度那边一反馈，不到五分钟就好了

**Comment by juangoubq** [comment]
我很想问并发量是多少，一个人的账号可以同时几个设备登录

**Reply by juangoubq** [reply]
求答复

**Reply by Namialways** [reply]
会员和普通用户六台，超级会员十台
